kroxylicious-wasm-sample-poc/src/main/resources/rust/compile.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+set -euxo pipefail
+# rustup target add wasm32-unknown-unknown
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+rm -f ${SCRIPT_DIR}/../wasm/*
+
+rustc fromto.rs --target=wasm32-unknown-unknown --crate-type=cdylib -C opt-level=0 -C debuginfo=0 -o ${SCRIPT_DIR}/../wasm/fromto.wasm
```

Comment:
We should call out the dependencies that need to be installed in this module's README.